apps/mull-api/src/app/auth/local.strategy.ts

```diff
@@ -0,0 +1,20 @@
+import { Strategy } from 'passport-local';
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { User } from '../entities/user.entity';
+
+@Injectable()
+export class LocalStrategy extends PassportStrategy(Strategy) {
```

Comment:
I think the goal was to have this be done in #30. Technically, this ticket was only considering registration, but for OAuth method I included logging in since it felt like it was blocking the flow in the backend.

Reply:
the local strategy is part of the ac for task 19. I don't mind doing it in #30 thou